website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -70,17 +70,17 @@ This POST endpoint requires a JSON object with the following properties as a req
+| `data.attributes.kind`                  | string         | `sentinel`                                    | The policy-as-code framework for the policy. Valid values are `sentinel` and `opa`.                                                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.query`                 | string         |                                               | The OPA query to run. Only valid for OPA policies.                                                                                                                                                                                                                                                                                                                                                                       |
+| `data.attributes.enforce`               | array\[object] |                                               | An array of enforcement configurations that map policy file paths to their enforcement modes. Policies support a single file, so this array should consist of a single element. For Sentinel, if the path in the enforcement map does not match the Sentinel policy (`<NAME>.sentinel`), then Terraform Cloud uses the default `hard-mandatory` enforcement level. For OPA, the default enforcement level is `advisory`. |
+| `data.attributes.enforce[].path`        | string         |                                               | For Sentinel, must be `<NAME>.sentinel`, where `<NAME>` has the same value as `data.attributes.name`. For OPA, must be `<NAME>.rego`.                                                                                                                                                                                                                                                                                    |
+| `data.attributes.enforce[].mode`        | string         | `hard-mandatory`(Sentinel) , `advisory` (OPA) | The enforcement level of the policy. For Sentinel, valid values are `"hard-mandatory"`, `"soft-mandatory"`, and `"advisory"`. For OPA, Valid values are `"mandatory"`and `"advisory"`. Refer to [Managing Policies](/cloud-docs/policy-enforcement/manage-policies) for details.                                                                                                                                         |
```

Comment:
This has double quotes, too! should it? I feel like we might be inconsistent on this in our docs to be honest - can you check with the other engineers to see what the actual preference is?